setup.py

```diff
@@ -50,9 +50,9 @@ def get_console_scripts():
     return (
         ['{clif.cli} = virtualenvrunner.cli:{clif.function}'.format(
             clif=clif) for clif in get_clifunctions()] +
-        [('{clif.cli}{major}.{minor} = '
+        [('{clif.cli}{version} = '
           'virtualenvrunner.cli:{clif.function}{major}{minor}'.format(
-              major=v.major, minor=v.minor, clif=clif))
+             version=str(v), major=v.major, minor=v.minor, clif=clif))
```

Comment:
No need to use `str(v)` as `format` is calling that automatically unless some format is specified.